stackanetes/common/utils.py

```diff
@@ -21,3 +25,35 @@ def env(*args, **kwargs):
         if value:
             return value
     return kwargs.get('default', '')
+
+
+def get_kubectl_command(add_namespace=True):
```

Comment:
I am not sure we should enforce a namespace by default. Why not using K8S's default by default and let the operator decide whether he wants to deploy under another namespace?